bench-tps/src/cli.rs

```diff
@@ -242,7 +242,7 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
         )
         .arg(
             Arg::with_name("tx_count")
-                .long("tx_count")
+                .long("tx-count")
```

Comment:
Can you please preserve the original form as an alias? Decent chance there's a script out there with `tx_count` in it, and it would be impolite to break it.

Reply:
Done: 0fea0b7182